docs/developer/dependencies.md

```diff
@@ -12,15 +12,22 @@ The python requirements are defined in `requirements.in` and `requirements_dev.i
 (Re)Generate the requirements files:
 
 ```
-$ docker run --rm -v $(pwd):/code standards-lab_standards-lab pip-compile
+$ docker run --rm -v $(pwd):/code rhiaro/pip-tools <selected requirements.in file>
 ```
 
 Upgrade the requirements:
 
 ```
-$ docker-compose run --rm -v $(pwd):/code standards-lab-web pip-compile --upgrade <selected requirements.in file>
+$ docker run --rm -v $(pwd):/code rhiaro/pip-tools --upgrade <selected requirements.in file>
 ```
 
+To install updated dependencies (TODO: use a shared volume to make this faster):
+
+```
+$ docker-compose down
+$ docker-compose build
+$ docker-compose up
+```
```

Comment:
Should the changes on this file not be in another PR?

Reply:
It was faster to update the PR description than pull the commits out :)